modules/order/src/Form/OrderTypeForm.php

```diff
@@ -79,6 +79,33 @@ public function form(array $form, FormStateInterface $form_state) {
       '#field_suffix' => t('seconds'),
     ];
 
+    $form['emails'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Emails'),
```

Comment:
The fieldset is called "Emails" (plural) but the notice says "The email" (singular). I guess we should just rephrase the notice to say "Emails are sent in the HTML format."